docs/solutions/rhel74_image/README.md

```diff
@@ -0,0 +1,43 @@
+This example provides a method to generate a RHEL 7.4 image for use by both VM and BM shapes.
+There are several prerequisites for using this process:
+
+1. You MUST have a valid RedHat account with subscriptions available.  The TF template needs a 
+   RH Username and Password to allow you to temporarily subscribe the instance that is building the image 
+   and get access to the various RH repos.
+2. The template expects pre-configured VCNs and Subnets.  
+3. You need to have a bucket in the OCI object store that contains the RHEL 7.4 iso.  The bucket must be in 
+   your tenancy, and must be accessible by the user executing the build.
+4. The template uses filters that expect unique Compartment, VCN and Subnet names.
+5. The following must be specified in your shell environment (prefixed with TF_VAR_ of course:
```

Comment:
Not blocking, but most of our examples/solutions provide an env-vars file for setting all necessary variables. (That could also include the info being set in configuration.tf.)